packages/expansion-hub/src/open-rev-hub.ts

```diff
@@ -25,10 +30,49 @@ export async function openParentRevHub(serialNumber: string): Promise<ExpansionH
 
     let serial = openSerialMap.get(serialPortPath)!;
 
-    let parentHub = new ExpansionHubInternal();
+    let parentHub = new ExpansionHubInternal(true, serialNumber);
+
+    if(moduleAddress === undefined) {
+        let addresses = await RevHub.discoverRevHubs(serial);
+        moduleAddress = addresses.parentAddress;
+    }
+
+    await parentHub.open(serial, moduleAddress);
+    await parentHub.queryInterface("DEKA");
+
+    if(parentHub.isParent()) {
+        return parentHub;
+    } else {
+        throw new Error(`Hub at ${serialNumber} with moduleAddress ${moduleAddress} is not a parent`);
+    }
+}
+
+/**
+ * Opens a parent REV hub, given that you know its {@link serialNumber}, as well as all children.
+ * Determining the addresses of the parent and child hubs will take upwards of a second.
+ *
+ * @param serialNumber the serial number of the REV hub (should start with DQ)
```

Comment:
```suggestion
 * @param serialNumber the serial number of the REV hub (should start with "DQ")
```

Nice job making these function doc comments consistent.